src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -282,156 +298,143 @@ def __get_test_list_by_runtime(self):
             logger.error(f"The test list file does not exists! Please check the process output!")
             exit(-1)
 
-        test_list = list()
+        tests_dict = dict()
         with open(test_list_file_name) as test_list_file:
             test_suite = ""
             for test_name in test_list_file.read().split('\n'):
                 if "Running main() from" in test_name:
                     continue
                 if not ' ' in test_name:
-                    test_suite = test_name
+                    test_suite = test_name.replace(".", "")
                     continue
-                pos = test_name.find('#')
+                pos = test_name.find(' # ')
                 if pos > 0 or test_suite != "":
-                    real_test_name = test_suite + (test_name[2:pos-2] if pos > 0 else test_name[2:])
+                    real_test_name = test_suite + "." + (test_name[2:pos-1] if pos > 0 else test_name[2:])
                     if constants.DISABLED_PREFIX in real_test_name:
                         self._disabled_tests.append(real_test_name)
-                    else:
-                        test_list.append(f'"{self.__replace_restricted_symbols(real_test_name)}":')
+                    elif (test_unit == "test"):
```

Comment:
`()` is not required to use in python conditions

Reply:
fixed